stores/sql/mysql/chain.go

```diff
@@ -210,10 +211,31 @@ func (c chainUpdateTx) FileContractElement(fcid types.FileContractID) (types.V2F
 	return ssql.FileContractElement(c.ctx, c.tx, fcid)
 }
 
+func (c chainUpdateTx) IsKnownContract(fcid types.FileContractID) (bool, error) {
+	if c.known == nil {
+		c.known = make(map[types.FileContractID]bool)
+	}
+
+	if relevant, ok := c.known[fcid]; ok {
+		return relevant, nil
+	}
+
+	known, err := ssql.IsKnownContract(c.ctx, c.tx, fcid)
+	if err != nil {
```

Comment:
`chainUpdateTx` has a logger so we can log the `err` here

Reply:
Not sure why we'd need to log that particular error. It shouldn't happen unless something is wrong with your database and then it will panic.